pip/tests/test_qsharp.py

```diff
@@ -373,3 +375,99 @@ def test_target_profile_from_str_match_enum_values() -> None:
     assert qsharp.TargetProfile.from_str(str_value) == target_profile
     with pytest.raises(ValueError):
         qsharp.TargetProfile.from_str("Invalid")
+
+
+def test_callables_exposed_into_env() -> None:
+    qsharp.init()
+    qsharp.eval("function Four() : Int { 4 }")
+    assert qsharp.env.Four() == 4
```

Comment:
Is this syntax used to avoid confusion in determining if the call is Q# or Python native? My first thought was to declare the callable in current scope, but I can see that it might cause confusion.

Reply:
putting it in a well-known module would help avoid conflicts/shadowing of things in the Python environment. @minestarks and I are still debating on the module name itself and looking for ideas... `qsharp.env` was the first one but we are wondering if something like `qsharp.code` or `qsharp.my` or `qsharp.current` or something. Any thoughts on that?